docs/getting-started/concepts/permission.md

```diff
@@ -0,0 +1,143 @@
+# Permission
+
+## Overview
+
+The Feast permissions model allows to configure granular permission policies to all the resources defined in a feature store.
+
+The configured permissions are stored in the Feast registry and accessible through the CLI and the registry APIs.
+
+The permission authorization enforcement is performed when requests are executed through one of the Feast (Python) servers
+- The online feature server (REST)
+- The offline feature server (Arrow Flight)
+- The registry server (grpc)
+
+On the contrary, there is no permission enforcement when accessing the Feast API with a local provider.
+
+## Concepts
+
+The permission model is based on the following components:
+- A `resource` is a Feast object that we want to secure against unauthorized access.
+  - We assume that the resource has a `name` attribute and optional dictionary of associated key-value `tags`.
+- An `action` is a logical operation executed on the secured resource, like:
+  - `create`: Create an instance.
+  - `read`: Access the instance state.
```

Comment:
It is not obvious to me what the difference is between `read` and `query`. What do we mean "access" the instance state? Can you elaborate?

Reply:
I agree it needs to be clarified a bit more.  query is able to query all the feature view meta data(such as name) where as read will allow the user to read actual feature view data from online_store. I will update the PR.

Reply:
I think we'll need to revisit that naming convention to make it more obvious.

Reply:
@tokoko what do you think here?

Reply:
This might have been my suggestion :smile: `read` refers to R in CRUD and `query` is the next best name for data retrieval. Maybe we can change `read` to `describe` and we will end up with `describe` and `query`. Should be much less ambiguous.

Reply:
Yes, @tokoko is definitely the ghost writer behind this design 😉 
We can rename it with low effort but I would keep the utility constant CRUD as is, w/o renaming it to `CDUD`

Reply:
yeah, I'm fine with keeping `CRUD` intact. We can make the change to `describe` if @franciscojavierarceo is also in agreement.

Reply:
Yeah I do think `describe` is more intuitive. I definitely appreciate the `query` language but I think following `CRUD` is a good enough convention that it'd be better to follow that.

Reply:
Addressing this changein next commit